test/hato/middleware_test.clj

```diff
@@ -6,6 +6,17 @@
             GZIPOutputStream)
            (java.io ByteArrayOutputStream ByteArrayInputStream InputStream)))
 
+(defmulti as-string type)
```

Comment:
Use https://github.com/ztellman/byte-streams ?

Reply:
way more dependencies, including manifold. That's a lot for just some conversions.

Reply:
Fair enough. :)